README.md

```diff
@@ -62,8 +62,18 @@ The PyMarkdown project has the following advantages:
 
 ## Note
 
-This project is currently in pre-release, and some of these documented things
-may not work 100% as advertised until after the initial release.
+This project is currently in beta, and some of these documented things
+may not work 100% as advertised until after the final release.  However,
+everything should be close enough to done that if you find something missing,
+please let us know.
+
+## GitHub Pre Commit Hooks
```

Comment:
```suggestion
## Git Pre-Commit Hooks
```

pre-commit and git pre-commit hooks are not in any way tied to GitHub, they works with any git repository, so the references should be "Git Pre-Commit [...]", not "GitHub Pre-Commit [...]" everywhere in this PR.